web/skins/classic/includes/control_functions.php

```diff
@@ -18,344 +18,389 @@
 // Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 // 
 
-function getControlCommands( $monitor ) {
-  $cmds = array();
+function getControlCommands( $monitor )
+{
```

Comment:
There seems to be a lot of unrelated style changes in this file. Can you revert them?

Reply:
My apologies. Something is really screwed up with those commits. I'll revert them and recommit the fixes.

Reply:
Okay. This problem was due to my convoluted workflow. My working branch was based off of an older version of master and so effectively reverted a number of changes made in master since the older version. I've opened pull request #2554 for corrected patches.

Thanks for drawing my attention to a real mess!